src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -227,7 +227,7 @@ void InitializeShortcutBar()
         {
             ShortcutToolbar shortcutBar = new ShortcutToolbar(this.dynamoViewModel.Model.UpdateManager);
             shortcutBar.Name = "ShortcutToolbar";
-
+            Library.Text = Properties.Resources.LibraryText;
```

Comment:
`TextBox` *is* UI control. Take a look at other textboxs and find out how they bind to strings in resource.